arches/app/media/js/views/components/workflows/new-multi-tile-step.js

```diff
@@ -0,0 +1,73 @@
+define([
```

Comment:
@whatisgalen looks like there are a number of unused imports here which should be removed.  It looks like only `'knockout'`, `'views/components/workflows/new-tile-step'`, and `'viewmodels/alert'` are actually being used in this module.